src/polymer-language-server.ts

```diff
@@ -106,12 +106,46 @@ connection.onInitialize((params): InitializeResult => {
   };
 });
 
+async function mapAllInitialDocuments(editorService: EditorService) {

```

Comment:
That way we can be sure that editorService is defined. Noticing that we have tons of checks for undefined checks for editorService was part of what inspired #62